pkg/services/ngalert/metrics/remote_writer.go

```diff
@@ -0,0 +1,30 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type RemoteWriter struct {
+	WritesTotal   *prometheus.CounterVec
+	WriteDuration *prometheus.HistogramVec
+}
+
+func NewRemoteWriterMetrics(r prometheus.Registerer) *RemoteWriter {
+	return &RemoteWriter{
+		WritesTotal: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "remote_writer_writes_total",
+			Help:      "The total number of remote writes attempted.",
+		}, []string{"org", "backend", "status_code"}),
+		WriteDuration: promauto.With(r).NewHistogramVec(
+			prometheus.HistogramOpts{
+				Namespace: Namespace,
+				Subsystem: Subsystem,
+				Name:      "remote_writer_write_duration_seconds",
+				Help:      "Histogram of remote write durations.",
+				Buckets:   prometheus.DefBuckets,
+			}, []string{"org", "backend", "status_code"}),
```

Comment:
We got status_code already on the counter - and we usually don't care at all about write duration of non-successful requests, we usually just care about general slowness. So, I think we can drop this label.

Histograms are pretty expensive, it's like a metric for each bucket in DefBuckets plus a sum. So really, this is like 12 or 13 metrics in one, multiplied by every single status code that we ever respond with, ends up with some pretty explosive cardinality.